### DISCUSSION TITLE
STARTTLS NOT WORKING

### DISCUSSION THREAD

**Discussion by 5656wwed** [postalserver/postal#1978] [Help with using Postal]
Not sent STARTTLS entenston not supported by server

PLEASE what can I do to resolve this issue problem

**Comment by uhlhosting** [comment]
@5656wwed please check the Discussions there are several solved cases like this. And provide details of what you have done until now. We are not wizards here to guess, from your statement not much we could help anyhow.

0 Check previous discussion on TLS/SSL issues. 
1 Provide steps you did to get it working.
2 What errors you got?

**Comment by willpower232** [comment]
You will need to turn this feature on in the config file. This has to be a manual process as you have to provide a certificate file.
